content/2017-04-27_mi-pyt.md

```diff
@@ -0,0 +1,183 @@
+
+<span style="color: #00aa00">19/04/2017 13:41:22</span><span style="color: #E850A8"> (https://twitter.com/HonzaKral/statuses/854691392865669120)</span>
+<span style="color: #0000aa"></span><span style="font-weight: bold; color: #0000aa">Honza Král</span><span style="color: #0000aa"> [HonzaKral]</span>: <span style="color: #aa5500"></span><span style="font-weight: bold; color: #aa5500">@jschovan</span> <span style="color: #aa5500"></span><span style="font-weight: bold; color: #aa5500">@naPyvo</span> <span style="color: #aa5500"></span><span style="font-weight: bold; color: #aa5500">@kvbik</span> <span style="color: #aa5500"></span><span style="font-weight: bold; color: #aa5500">@tylersiprova</span> <span style="color: #aa5500"></span><span style="font-weight: bold; color: #aa5500">@honzajavorek</span> <span style="color: #aa5500"></span><span style="font-weight: bold; color: #aa5500">@aleszoulek</span> <span style="color: #aa5500"></span><span style="font-weight: bold; color: #aa5500">@EnCuKou</span> <span style="color: #aa5500"></span><span style="font-weight: bold; color: #aa5500">@almadcz</span> <span style="color: #aa5500"></span><span style="font-weight: bold; color: #aa5500">@benAbraham</span> Ja taky budu
+
+<span style="color: #00aa00">19/04/2017 13:41:54</span><span style="color: #E850A8"> (https://twitter.com/aleszoulek/statuses/854691529876811777)</span>
+<span style="color: #0000aa"></span><span style="font-weight: bold; color: #0000aa">Ales Zoulek</span><span style="color: #0000aa"> [aleszoulek]</span>: <span style="color: #aa5500"></span><span style="font-weight: bold; color: #aa5500">@HonzaKral</span> <span style="color: #aa5500"></span><span style="font-weight: bold; color: #aa5500">@jschovan</span>
+</pre>
+<figcaption class="figure-caption" style="text-align: center">
+TwitterWall do terminálu, geekův sen (převedeno na HTML z výstupu aplikace Marka Suchánka)</figcaption></figure>
```

Comment:
Jsem líný dělat checkout a pouštět `blog preview`, abych viděl, jak se tahle věc zobrazí, ale věřím, že jste to udělali vy dva 😛